### ISSUE TITLE
[Accessibility]: Case Study floating box not mobile friendly

### ISSUE THREAD

**Issue by st0nebraker** [sourcegraph/about#5628]
### Audit type

Mobile device navigation

### User journey audit issue

#5467

### Problem description

The new floating box design on Case Studies does not keep it's text content wrapped on small mobile screens (down to `320px`)

### Expected behavior

All our layouts should be responsive down to `320px` screen widths
